ttnn/cpp/ttnn/operations/pool/upsample/upsample_pybind.hpp

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+#include "upsample.hpp"
```

Comment:
please split into .hpp and .cpp 
reduce headers here, these includes are heavy.

see ccl/all_gather op as example.